forc-fmt/src/main.rs

```diff
@@ -30,13 +31,17 @@ pub struct App {
     pub path: Option<String>,
 }
 
-fn main() -> Result<()> {
+fn main() {
+    init_tracing_subscriber();
     let app = App::parse();
     let dir = match app.path.clone() {
         Some(p) => PathBuf::from(p),
-        None => std::env::current_dir()?,
+        None => std::env::current_dir().expect("cannot get current dir"),
     };
-    format_pkg_at_dir(app, &dir)
+    if let Err(err) = format_pkg_at_dir(app, &dir) {
+        error!("Error: {:?}", err);
+        std::process::exit(1);
+    }
```

Comment:
Instead of changing `current_dir` to use `expect` on the result, maybe we could introduce a `run` function that still returns `Result`, something like this:

```rust
fn run() -> Result<()> {
    let app = App::parse();
    let dir = match app.path.clone() {
        Some(p) => PathBuf::from(p),
        None => std::env::current_dir()?,
    };
    format_pkg_at_dir(app, &dir)
}

fn main() {
    init_tracing_subscriber();
    if let Err(err) = run() {
        error!("Error: {:?}", err);
        std::process::exit(1);
    }
}
```

Reply:
That looks much better! Thanks